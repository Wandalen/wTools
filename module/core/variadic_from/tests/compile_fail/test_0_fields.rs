// tests/compile_fail/test_0_fields.rs

#[ allow( dead_code ) ]
#[ derive( variadic_from::VariadicFrom ) ]
struct Test0FieldsNamed {}