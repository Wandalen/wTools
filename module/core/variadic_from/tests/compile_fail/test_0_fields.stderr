error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compile_fail/test_0_fields.rs:5:27
  |
5 | struct Test0FieldsNamed {}
  |                           ^ consider adding a `main` function to `$DIR/tests/compile_fail/test_0_fields.rs`
